docs/index.md

```diff
@@ -0,0 +1,109 @@
+# Perseo Context-Aware CEP
```

Comment:
This file seems to be a copy of README.md... I think it shouldn't duplicate content. Two alternatives:

* To be just an introduction to the contents of the dir, for instance: https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/admin/index.md
* To be a ToC of the contents of the directory, for instance: https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/admin/README.md

Reply:
It should be an abstract for the document only.

> Perseo is a Complex Event Processing (CEP) software designed to be fully _NGSIv2_-compliant. It uses NGSIv2 as the
communication protocol for events, and thus, Perseo is able to seamless and jointly work with _context brokers_ such as
[Orion](https://github.com/telefonicaid/fiware-orion).
>

Readthedocs will create a sidebar toc, GitHub will list all other documents

Reply:
Ok. abstract added in docs/index.md. Fixed in ef4582bceac76559b84a45f17bef934c90ce6529

Reply:
Not sure if the doc is ok or it stills replicating too much information for master README.md... @jason-fox what do you think?

Reply:
The `index.md` in the documentation should usually just be an [abstract](https://en.wikipedia.org/wiki/Abstract_(summary)) - typically Repeating part of the `readme` is fine IMHO since it is likely a reader may be experiencing Perseo as a user only (rather than a developer) and therefore would be doing it without looking at the `readme`.

An intro just needs to elevator pitch what this component does (and for tech documents there is no need to use marketing hyperbole). The one from [Orion](https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/index.md) hits the sweet spot for me for length.  Something like [Keyrock](https://github.com/ging/fiware-idm/blob/master/doc/index.md) is OK, but the breaking changes part is misplaced and should be an appendix IMHO.

Reply:
@cblanco does feedback provided by @jason-fox addressed at the end?